src/components/Button/__tests__/Button.test.tsx

```diff
@@ -61,7 +73,7 @@ describe('Button', () => {
 
   it('shows ButtonContainer ButtonVariant.outline', async () => {
     const { container, getByTestId } = render(
-      <Button variant={variants.outline} onClick={() => {}} />,
+      <Button dataTestId={testId} variant={variants.outline} onClick={() => {}} />,
```

Comment:
This is not a comment for changes you should make (just a mini-rant), but there's a trend in testing in libraries like this (and what this testing library was designed for) where you pretend like you're doing an integration test and do stuff like "Find the text that says "Click me" and click on that element and make sure the thing works as intended. 

I'd love to see us using data-test-ids as little as possible, since our users aren't clicking on elements with a particular test-id. I know this way of doing things was already there so I'm not expecting or asking for a change but just wanted to call that out

cc @aVileBroker

Reply:
Yeah I'm using dataTestId here more so that we can check if we're clicking blurring to another element which isn't part of _this_ dropdown. I don't like using name, because it means it would need to be a required field or be a nanoid random defualt - but name attributes do actually mean things in forms so I don't like just having it be randomized. This is also why I made name be optional and default to "dropdown.

I'm fine with using a different attribute than dataTestId though. (I'm not even using dataTestId in the tests lol)